### PR TITLE
Docs: update index.md based on UX feedback

### PR DESCRIPTION
**Pull request by brian-intel**
_The PR review is to check for sustainability and correctness.  Sustainability is actually more business critical as correctness is largely tested into the code over time.   Its useful to keep in mind that SW often outlives the HW it was written for and engineers move from job to job so it is critical that code developed for Intel be supportable across many years.  It is up to the submitter and reviewer to look at the code from a perspective of what if we have to debug this 3 years from now after the author is no longer available and defect databases have been lost.  Yes, that happens all the time when we are working with time scales of more than 2 years.  When reviewing your code it is important to look at it from this perspective._

Author Mandatory (to be filled by PR Author/Submitter)
------------------------------------------------------
- Developer who submits the Pull Request for merge is required to mark the checklist below as applicable for the PR changes submitted.  
- Those checklist items which are not marked are considered as not applicable for the PR change.  
- Items marked with an asterisk suffix are mandatory items to check and if not marked will be treated as non-compliant pull requests by the developers for Inner Source Development Model (ISDM) compliance

### PULL DESCRIPTION
Docs: update index.md based on UX feedback

### REFERENCES
Reference URL for issue tracking (Github): **\<URL to be filled>**

- [x] Added label to the Pull Request for easier discoverability and search

### CODE MAINTAINABILITY
- [x] **_Commit Message meets guidelines as indicated in the URL\*_**
	- https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] **_Every commit is a single defect fix and does not mix feature addition or changes\*_**
- [ ] Added required new tests relevant to the changes
	- [ ] PR contains URL links to functional tests executed with the new tests 
- [x] Updated Documentation as relevant to the changes
- [ ] Updated Build steps/commands changes as relevant
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide description)
- [ ] Specific instructions or information for code reviewers (If any):

Maintainer Mandatory (to be filled by PR Reviewer/Approving Maintainer)
-----------------------------------------------------------------------
- Maintainer who approve the Pull Request for merge are required to mark the checklist below as appropriate for the PR change reviewed as key proof of attestation indicating reasons for merge. 
- Those checklist items which are not marked are considered as not applicable for the PR change. 
- Items marked with an asterisk suffix are mandatory items to check.

### QUALITY CHECKS
- [x] Architectural and Design Fit
- [x] **_Quality of code (At least one should be checked as applicable)\*_**
	- [x] Commit Message meets guidelines
	- [ ] PR changes adhere to industry practices and standards
	- [ ] Adopted domain specific coding standards ([SBL Coding Standards](https://github.com/tianocore-docs/Docs/raw/master/Specifications/CCS_2_1_Draft.pdf))
	- [ ] Error and exception code paths implemented correctly
	- [ ] Code reviewed for domain or language specific anti-patterns
	- [ ] Code is adequately commented
	- [x] Code copyright is correct
	- [ ] Tracing output are minimized and logic
	- [ ] Confusing logic is explained in comments
	- [ ] Commit comment can be used to design a new test case for the changes
- [ ] **_Test coverage shows adequate coverage with required CI functional tests pass on all supported platforms\*_**
- [ ] **_Static code scan report shows zero critical issues\*_**

### CODE REVIEW IMPACT
- Summary of Defects Detected in Code Review: **\<%P1*xx,P2*xx,P3*xx,P4*xx%>** \
Note P1/P2/P3/P4 denotes severity of defects found (Showstopper/High/Medium/Low) and xx denotes number of defects found

# _Code must act as a teacher for future developers_

### CODE REVIEW

File: docs_src/index.md [hunk 1]
```diff
@@ -1,34 +1,25 @@
-# Vision Self Checkout
+# Automated Self Checkout
 
-## Introduction
-This guide helps you build and run the Vision Self Checkout solution.
+## Overview
 
-Upon completing the steps in this guide, you will be ready to run and benchmark pipelines on different hardware setups.
+The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy Automated self-checkout using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured Automated self checkout pipeline that is optimized for Intel® hardware. Also included in is a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per vision stream.
```
Comment (jim-wang-intel) [inline docs_src/index.md:5]: Some re-wording suggestions

```suggestion
The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy self-checkout reference solution using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipelines that are optimized for Intel® hardware.  Moreover, there also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per video stream.
```

File: docs_src/index.md [hunk 2]
```diff
@@ -1,34 +1,25 @@
...
-[![Vision Self Checkout Diagram](./images/vision-checkout-1.0.png)](./images/vision-checkout-1.0.png)
+- [Ubuntu LTS Boot Device](https://ubuntu.com/tutorials/install-ubuntu-desktop#3-create-a-bootable-usb-stick)
+- [Docker](https://docs.docker.com/engine/install/ubuntu/)
+- [Git](https://git-scm.com/download/linux)
 
-### Prerequisites
+## Platform Installation and Pipeline Setup
 
-The following items are required to build the Vision Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.
+first verify that your platform is included on the [supported platform list](./platforms.md)
```
Comment (jim-wang-intel) [inline docs_src/index.md:19]:
```suggestion
First verify that your platform is included on the [supported platform list](./platforms.md)
```

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: docs_src/index.md [hunk 3]
```diff
@@ -1,34 +1,25 @@
-# Vision Self Checkout
+# Automated Self Checkout
 
-## Introduction
-This guide helps you build and run the Vision Self Checkout solution.
+## Overview
 
-Upon completing the steps in this guide, you will be ready to run and benchmark pipelines on different hardware setups.
+The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms. Together the reference solution will help evaluate your required hardware to minimize the cost per workload.
```
Comment (ejlee3) [inline docs_src/index.md:5]: are you currently providing components to build/deploy for the self-checkout use case? or is it mostly benchmarking?

Comment (ejlee3) [inline docs_src/index.md:5]:
```suggestion
The Intel® Automated Self-Checkout Retail Reference Software provides critical components required to build and deploy a self-checkout use case using Intel® hardware, software, and other open-source software. This reference implementation provides a pre-configured automated self-checkout pipeline that is optimized for Intel® hardware.  The reference solution also includes a set of benchmarking tools to evaluate the workload on different hardware platforms in order to minimize the cost per workload.
```

File: docs_src/index.md [hunk 4]
```diff
@@ -1,34 +1,25 @@
...
-A set of branchmarking tools have been provided to demonstrate the performance of the pipelines on Intel hardware. Once you have completed the [pipeline setup steps](./pipelinesetup.md) you will be able to run benchmark by following these [steps](./benchmark.md) 
+The following items are required to build the Automated Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.
 
-[![Vision Self Checkout Diagram](./images/vision-checkout-1.0.png)](./images/vision-checkout-1.0.png)
+- [Ubuntu LTS Boot Device](https://ubuntu.com/tutorials/install-ubuntu-desktop#3-create-a-bootable-usb-stick)
+- [Docker](https://docs.docker.com/engine/install/ubuntu/)
+- [Git](https://git-scm.com/download/linux)
 
-### Prerequisites
+## Platform Installation and Pipeline Setup
```
Comment (ejlee3) [inline docs_src/index.md:17]: This section is missing the pipeline setup link

File: docs_src/index.md [hunk 5]
```diff
@@ -1,34 +1,25 @@
...
 
-The following items are required to build the Vision Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.
+First verify that your platform is included on the [supported platform list](./platforms.md)
```
Comment (ejlee3) [inline docs_src/index.md:19]:
```suggestion
First verify that your platform is included on the [supported platform list](./platforms.md).
```

File: docs_src/index.md (hunk 3 above)
Comment (brian-intel) [reply docs_src/index.md:5]: It is a mix of both but most of it is tied to the benchmarking scripts

File: docs_src/index.md (hunk 4 above)
Comment (brian-intel) [reply docs_src/index.md]: Updated to only point to the platform information which is the logical next step

File: docs_src/index.md [hunk 6]
```diff
@@ -1,34 +1,25 @@
...
-### Prerequisites
+## Platform Installation
 
-The following items are required to build the Vision Self Checkout solution. You will need to follow the guide that matches your specific hardware setup.
+First verify that your platform is included on the [supported platform list](./platforms.md)
 
-- Ubuntu LTS Boot Device
-- Docker
-- GIT
+Once verified the setup steps for each supported hardware platform can be found [HERE](./hardwaresetup.md).
```
Comment (vli11) [inline docs_src/index.md:21]: Replace "Once verified the setup steps..." with "Verify that the setup steps..."

**Review by vli11 — APPROVED** [review]
LGTM

**Pull request merged**